src/control/character_controller.rs

```diff
@@ -163,12 +163,13 @@ impl Default for KinematicCharacterController {
             max_slope_climb_angle: Real::frac_pi_4(),
             min_slope_slide_angle: Real::frac_pi_4(),
             snap_to_ground: Some(CharacterLength::Relative(0.2)),
-            normal_nudge_factor: 1.0e-4,
+            normal_nudge_factor: 1.0e-5,
```

Comment:
This was messing up the grounded status on the test scene ; if that's a bad default, more explaination and/or a test case for this default should be added.